app/router.js

```diff
@@ -6,6 +6,10 @@ const Router = Ember.Router.extend({
 });
 
 Router.map(function() {
+  this.route('list', function() {
```

Comment:
Add a custom path of `/` so that the lists are seen immediately available when loading the root of the app.